pkg/agent/memberlist/cluster.go

```diff
@@ -454,9 +454,12 @@ func (c *Cluster) syncConsistentHash(eipName string) error {
 		}
 		consistentHashMap := NewNodeConsistentHashMap()
 		consistentHashMap.Add(aliveAndMatchedNodes...)
+		allNodesConsistentHashMap := NewNodeConsistentHashMap()
+		allNodesConsistentHashMap.Add(aliveNodes.UnsortedList()...)
 		c.consistentHashRWMutex.Lock()
 		defer c.consistentHashRWMutex.Unlock()
 		c.consistentHashMap[eip.Name] = consistentHashMap
+		c.consistentHashMap[""] = allNodesConsistentHashMap
```

Comment:
Updating the allNodes hashMap here would introduce duplicate handlings. I think we can just update it in `handleClusterNodeEvents`, without recalculating all nodes, and just use the hashMap's Add or Remove method to update one node which is notified by the event.